core/src/main/java/pl/agh/edu/engine/opinion/OpinionHandler.java

```diff
@@ -1,17 +1,74 @@
 package pl.agh.edu.engine.opinion;
 
+import static java.math.BigDecimal.ONE;
+import static java.math.BigDecimal.ZERO;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
+import java.util.OptionalDouble;
+import java.util.function.Function;
 
+import pl.agh.edu.data.loader.JSONOpinionDataLoader;
+import pl.agh.edu.engine.time.Time;
+import pl.agh.edu.engine.time.TimeCommandExecutor;
+import pl.agh.edu.engine.time.command.TimeCommand;
 import pl.agh.edu.utils.RandomUtils;
 
 public class OpinionHandler {
+	private static BigDecimal opinionModifier = new BigDecimal("0.1");
```

Comment:
czy tego by też do JSON nie wrzucić?

Reply:
Bo to rozumiem że jest startowy modyfikator

Reply:
wg mnie nie ma po co tego tam wsadzać